EXTRACTOR_SERVICE/.env

```diff
@@ -1,4 +1,4 @@
-EXTRACTOR_ROOT=D:/Prog101/AT14/AT14-SERVICES/EXTRACTOR_SERVICE/
+EXTRACTOR_ROOT=C:/Users/serqi/Desktop/Fundacion/proga_101/project/AT14-SERVICES/EXTRACTOR_SERVICE
 EXTRACTOR_APP=http://localhost
-EXTRACTOR_PORT=3000
+EXTRACTOR_PORT=1000
```

Comment:
Why the port 1000 is for Extractor Service

Reply:
becasue 3000 is ui